bool detection_flag
float32 actMinLaserValue
float32 turnSpeed
---
bool success
int32 goal